code/modules/mob/living/damage_procs.dm

```diff
@@ -167,19 +167,21 @@
 
 /mob/living/proc/adjustOxyLoss(amount, updating_health = TRUE, forced = FALSE)
 	if(!forced && (status_flags & GODMODE))
-		return FALSE
+		return
+	. = oxyloss
 	oxyloss = clamp((oxyloss + (amount * CONFIG_GET(number/damage_multiplier))), 0, maxHealth * 2)
 	if(updating_health)
 		updatehealth()
-	return amount

```

Comment:
For the record, this is not how it used to work but it does not appear the return value of this proc was actually used anywhere.
apply_damage_type relays the return value but it also isn't used anywhere else.
Somewhat of a shame that this isn't consistent with how the other DamageLoss procs do it.

Reply:
I did notice that and did check for every call to see if it was ever used.
Ultimately I considered that returning the previous state, or `null` in case of failure, was more useful than returning the passed input value and you could still check how much damage was effectively applied by comparing the new state with the returned old one.